.circleci/config.yml

```diff
@@ -94,8 +94,7 @@ jobs:
           command: |
             if [ -n "${npm_TOKEN}" ] && [ "${CIRCLE_BRANCH}" == "master" ]; then
               echo "//registry.npmjs.org/:_authToken=${npm_TOKEN}" > ~/project/.npmrc
-              npm version prerelease --preid="${CIRCLE_BUILD_NUM}.${CIRCLE_BRANCH}.`git rev-parse --short HEAD`" --no-git-tag-version
-              npm publish --tag current
+              yarn lerna publish --canary --preid "${CIRCLE_BUILD_NUM}.${CIRCLE_BRANCH}" --yes
```

Comment:
just something to think about in the future - not absolutely sure that we should publish _all_ the packages after each build - actually, even now it creates lots of versions (for `react` package) that we are not consuming. I would see that there will be some trigger logic there (that might rely in the commit message) that would be required to express _explicit_ intent to publish package - and, otherwise, it shouldn't be published at all.